pdns/recursordist/html/local-2022.js

```diff
@@ -1,13 +1,44 @@
 "use strict";
 
-// var moment= require('moment');
+const fetchConfig = {
+    baseURL: window.location,
+    mode: 'same-origin',
+    headers: {'Accept': 'application/json'},
+};
+/*
+// Useful for development of the embedded webserver files.
+const fetchConfig = {
```

Comment:
This commented snippet can be confusing. Can you remove it and set `X-API-Key` by default?

Reply:
no, that will not work. in a production build, x-api-key should not be set - its only useful if the html/js files are used "unembedded".

Reply:
You're right - my bad!